### DISCUSSION TITLE
Power Platform methods?

### DISCUSSION THREAD

**Discussion by Gareth064** [pnp/pnpcore#1108] [General]
Hey, just a random one. I may have a requirement to start communicating with Flows in the Power Platform to read their run history, resubmit them etc, and I noticed there is a PnP Cli/M365 library that can do this.

Will such functionality make its way to the PnPCore SDK?

**Comment by jansenbe** [comment]
@Gareth064 : we indeed don't have that today, but we're open to extend in that direction if someone can help with that.
